### ISSUE TITLE
PYTHONPATH cleanup

### ISSUE THREAD

**Issue by yazug** [redhat-openstack/easyfix#18]
as identified by chandankumar

<chandankumar> jschlueter: export pythonpath no longer needed

python-oslo-context .spec file still has export PYTHONPATH in the doc generation section.

there may be more packages with similar packaging differences

## What to improve:

1. Clone tripleo-validations-distgit spec file git clone https://github.com/rdo-packages/glance-distgit
2. Go to glance-distgit and open  openstack-glance.spec in editor.
3. You can find PYTHONPATH in the build section of the in spec file.
4. We need to remove those lines from spec file (including relevant comments to PYTHONPATH).
5. Once done! Make changes, add and commit it and send a Gerrit Review.
6. Please cross [checklist](https://github.com/redhat-openstack/easyfix#checklist-for-sending-your-first-rdo-gerrit-review)
   before sending Gerrit Review.

## Below is the list of packages which need same fixes:

- [x] [barbicanclient](https://github.com/rdo-packages/barbicanclient-distgit)
- [X] [ceilometerclient](https://github.com/rdo-packages/ceilometerclient-distgit)

- [x] [cloudkitty](https://github.com/rdo-packages/cloudkitty-distgit)
- [x] [django_openstack_auth](https://github.com/rdo-packages/django_openstack_auth-distgit)
- [x] [glanceclient](https://github.com/rdo-packages/glanceclient-distgit)
- [x] [glance](https://github.com/rdo-packages/glance-distgit)
- [x] [gnocchiclient](https://github.com/rdo-packages/gnocchiclient-distgit)
- [x] [keystone](https://github.com/rdo-packages/keystone-distgit)
- [x] [manila](https://github.com/rdo-packages/manila-distgit)
- [x] [neutronclient](https://github.com/rdo-packages/neutronclient-distgit)
- [x] [nova](https://github.com/rdo-packages/nova-distgit)
- [x] [oslo-context](https://github.com/rdo-packages/oslo-context-distgit)
- [x] [oslo-vmware](https://github.com/rdo-packages/oslo-vmware-distgit)
- [x] [os-testr](https://github.com/rdo-packages/os-testr-distgit)
- [x] [sahara](https://github.com/rdo-packages/sahara-distgit)
- [X] [swiftclient](https://github.com/rdo-packages/swiftclient-distgit)

## Mentors

1. jschlueter (yazug)

**Comment by yazug** [comment]
also openstack-glance

**Comment by OGtrilliams** [comment]
glance-distgit - https://review.rdoproject.org/r/8094
glanceclient-distgit - https://review.rdoproject.org/r/8097
cloudkitty-distgit - https://review.rdoproject.org/r/8095
barbicanclient-distgit - https://review.rdoproject.org/r/8096

**Comment by snecklifter** [comment]
@yazug @chkumar246 @OGtrilliams the remaining patches are done

**Comment by snecklifter** [comment]
All patches are merged except for some which specifically require PYTHONPATH (Gnocchi).

**Comment by yazug** [comment]
https://review.rdoproject.org/r/#/q/branch:rpm-master+topic:easyfix/18+status:open still shows 3 open

**Comment by snecklifter** [comment]
@yazug thanks, the glance one needs abandoning as the fix already went in, the other two I have updated.

**Comment by snecklifter** [comment]
Just waiting on +W at https://review.rdoproject.org/r/#/c/8095/

**Comment by snecklifter** [comment]
Hooray. @yazug happy to close this one?

**Comment by yazug** [comment]
looks good and thanks!
